snmp/metadata.csv

```diff
@@ -1,2 +1,166 @@
+snmp.adapterRxMulticast,count,,packet,,[Dell iDRAC] Total number of Multicast packets received.,0,snmp,
+snmp.adapterRxPackets,count,,packet,,[Dell iDRAC] Total packets received.,0,snmp,
+snmp.adapterTxBytes,count,,byte,,[Dell iDRAC] Total number of bytes transmitted.,0,snmp,
+snmp.adapterTxDropped,count,,packet,,[Dell iDRAC] Total number of transmit packets dropped due to successive collisions.,0,snmp,
+snmp.adapterTxErrors,count,,packet,,[Dell iDRAC] Total number of packets that failed to transmit.,0,snmp,
+snmp.adapterTxPackets,count,,packet,,[Dell iDRAC] Total packets transmitted.,0,snmp,
+snmp.cefcFRUCurrent,gauge,,,,[Cisco c3850] [Cisco Nexus] Current supplied by the FRU (positive values) or current required to operate the FRU (negative values).,0,snmp,
+snmp.cefcFRUPowerAdminStatus,gauge,,,,[Cisco c3850] [Cisco Nexus]  Administratively desired FRU power state.,0,snmp,
+snmp.cefcFRUPowerOperStatus,gauge,,,,[Cisco c3850] [Cisco Nexus]  Operational FRU power state.,0,snmp,
+snmp.cieIfInputQueueDrops,gauge,,packet,,[Cisco c3850] The number of input packets dropped.,0,snmp,
+snmp.cieIfLastInTime,gauge,,millisecond,,[Cisco c3850] The elapsed time in milliseconds since the last protocol input packet was received.,0,snmp,
+snmp.cieIfLastOutTime,gauge,,millisecond,,[Cisco c3850] The elapsed time in milliseconds since the last protocol output packet was transmitted.,0,snmp,
+snmp.cieIfOutputQueueDrops,gauge,,packet,,[Cisco c3850] The number of output packets dropped by the interface even though no error was detected to prevent them being transmitted.,0,snmp,
+snmp.cieIfResetCount,count,,,,[Cisco c3850] [Cisco Nexus]  The number of times the interface was internally reset and brought up.,0,snmp,
+snmp.cpmCPUMemoryFree,gauge,,,,[Cisco c3850] [Cisco Nexus] The overall CPU wide system memory currently free.,0,snmp,
+snmp.cpmCPUMemoryUsed,gauge,,,,[Cisco c3850] [Cisco Nexus]  The overall CPU wide system memory currently in use.,0,snmp,
+snmp.cpmCPUTotalMonIntervalValue,gauge,,percent,,[Cisco c3850] [Cisco Nexus]  The overall CPU busy percentage in the last cpmCPUMonInterval period.,0,snmp,
+snmp.devClientCount,gauge,,,,[Cisco Meraki] The number of clients currently associated with the device.,0,snmp,
+snmp.devInterfaceRecvBytes,gauge,,byte,,[Cisco Meraki] The number of bytes received on this interface.,0,snmp,
+snmp.devInterfaceRecvPkts,gauge,,packet,,[Cisco Meraki] The number of packets received on this interface.,0,snmp,
+snmp.devInterfaceSentBytes,gauge,,byte,,[Cisco Meraki] The number of bytes sent on this interface.,0,snmp,
+snmp.devInterfaceSentPkts,gauge,,packet,,[Cisco Meraki] The number of packets sent on this interface.,0,snmp,
+snmp.devStatus,gauge,,,,[Cisco Meraki] The status of the device’s connection to the Meraki Cloud Controller,0,snmp,
+snmp.discovered_devices_count,gauge,,device,,The total number of devices discovered.,0,snmp,
```

Comment:
Not sure we want to call right out right now since this is the counting metric we're planning on using for billing. cc @therve @florimondmanca what do you think? 

Also fwiw I think it's very hidden in the docs here, not sure we have a choice where/when this gets pulled in.

Reply:
we could put this one on top if need be, metrics are sorted alphabetically for now.

Reply:
Is it possible to remove it, just in case that metric changes/or we use something else?

Reply:
From a doc perspective there is no impact, so i would say yes. @therve @florimondmanca do you have any objection?

```suggestion
```

Reply:
I think we should keep it in the docs @l0k0ms @naaltman. Even if we don't use it for billing eventually, this metric makes sense regardless, as it provides a way for customers to monitor how much of their subnet has been successfully discovered.

Reply:
Very good point. Let's keep it